app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PostsController < InheritedResources::Base
+
+  private
+
+    def post_params
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.